### PR TITLE
Create Intro_sort.py

### PR DESCRIPTION
**Pull request by nehasangeetajha**
Fixes #2249 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Implementation of IntroSort in Python (corrected everything, marked in the previous PR)

#### Languages Used:
- Python 

#### Files Added:
- Intro_Sort.py

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by nehasangeetajha** [comment]
@somya-kapoor Here is the new PR with all changes told previously

**Comment by somya-kapoor** [comment]
@nehasangeetajha Please squash the commits

**Comment by nehasangeetajha** [comment]
I am not able to do. I will try again

**Comment by somya-kapoor** [comment]
For reference: http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request

**Comment by nehasangeetajha** [comment]
tried the same but its not able to squash

**Comment by raghav-dalmia** [comment]
Are you working on it?

**Comment by nehasangeetajha** [comment]
Actually not able to squash the commits

File: Intro_Sort/Intro_Sort.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+def introsort(list):
```
Comment (MastersAbh) [inline Intro_Sort/Intro_Sort.py:1]: please add problem statement at the top of file

**Comment by somya-kapoor** [comment]
Are you working on it?

**Comment by nehasangeetajha** [comment]
Not able to

**Comment by somya-kapoor** [comment]
So should I close this pr?

**Comment by nehasangeetajha** [comment]
Ya

**Pull request closed without merging**